lib/api/objectDelete.js

```diff
@@ -169,8 +170,14 @@ function objectDelete(authInfo, request, log, cb) {
                     result.versionId : versionIdUtils.encode(
                         result.versionId, config.versionIdEncodingType);
             }
+            const versioningSuspended = bucketMD
+                && bucketMD._versioningConfiguration
+                && bucketMD._versioningConfiguration.status === 'Suspended';
+            const hasByteLength = objectMD && objectMD['content-length'];
```

Comment:
```suggestion
            const hasByteLength = objectMD?.['content-length'];
```
I believe this is the syntax for null chaining by index, could be wrong